### DISCUSSION TITLE
How can I make an editable subscription via the cart?

### DISCUSSION THREAD

**Discussion by gsumo** [swellstores/community#208] [Help] [answered]
If I use a product bundle, it doesn't seem like I can remove or add a sku to it. If I add multiple subscriptions to the cart it seems to create separate subscriptions. Is there a way to create one subscription and add / remove products to it using the shopping cart interface? and then in the future using the UI change products in the subscription?

**Comment by sags95** [comment, marked as answer]
@gsumo If you want to use the typical cart flow: Add products to cart -> `swell.cart.submitOrder()`. Then by default this will create separate subscriptions for each subscription purchase_option in the cart/order. There is a away to create one subscription with separate items but this would require a different flow as it does not use the cart: https://developers.swell.is/frontend-api/subscriptions#create-a-new-subscription. If you're building a custom checkout then perhaps you can take in the cart items and convert them into the call I linked above to create a single subscription with these items?
